wagtail/embeds/oembed_providers.py

```diff
@@ -520,8 +520,9 @@
 
 
 spotify = {
-    "endpoint": "https://embed.spotify.com/oembed/",
+    "endpoint": "https://open.spotify.com/oembed",
     "urls": [
+        r"^https?://open\.spotify\.com/episode/7makk4oTQel546B0PZlDM5",
```

Comment:
This specific URL should not be added here. 
```suggestion
```

Reply:
ok